openapi.yaml

```
@@ -1225,6 +1245,16 @@ components:
         - id
         - question_id
         - user_traq_id
+    GetQuestionAnswers:
+      type: object
+      properties:
+        question_id:
+          type: integer
+        answers:
+          type: array
+          items:
+            user_traq_id:
+              type: string
```

もし`Answer`を使わずに`GetQuestionsAnswers`を使う必要があるのであれば、ここの`user_traq_id`は消したほうが良いです（`user_traq_id`のようなフィールド名を指定する必要があるのは`type: object`のときで、単純な`string`の配列には必要ないです）。このままだと型が正確に解釈されず、`[]string`ではなく`[]interface`になってしまいます……
```suggestion
            type: string
```

---

そうなんですね。いわれてみたら名前を二つ付けるのは無駄でした。修正します。ありがとう！！